### PR TITLE
Finished Desktop Version Design

### PR DESCRIPTION
**Pull request by Medini-Rajendra**
## In this project, the following things are done
 - Readme file updated
 - Checked for linter errors.

### CODE REVIEW

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑❌🛠**
Hello @Medini-Rajendra 👋🏽
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._👇🏽👀
- I can see your page is overflowing. Please kindly fix that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Medini-Rajendra** [comment]
I have correct the page overflowing in my code using "overflow-x: hidden" in body tags.

File: index.html [hunk 1]
```diff
@@ -14,164 +14,197 @@
...
+                <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
+                <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
+                <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
+                <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
+                <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+            </ul>
+        </div>
     </section>
-    <div class="grid-container">
+    <div class="grid-container grid">
```
Comment (Mihndim2020) [inline index.html:43]: - [OPTIONAL] Please make sure this section is aligned a per the design, your projects seem to be aligned more to the left.  Kindly make sure your margins and alignments are as in the design. See image below

***The design***
medD

File: index.html [hunk 2]
```diff
@@ -14,164 +14,197 @@
...
-            <div class="lang_section">
-                <h2>Languages</h2>
-                <i class="fa fa-angle-down"></i>
-            </div>
-            <div class="js_flex1">
-                <div class="js_flex">
-                    <img src='./images/Ellipse_1.svg' alt="js_logo">
-                    <h3>&nbsp;&nbsp;JavaScript</h3>
+    <div class="get_lang_flex">
+        <section class="container about_section">
```
Comment (Mihndim2020) [inline index.html:138]: - [ ] Please there is a margin between "About myself" section and the projects section. Please kindly fix that. See image below;

***The design***
medD2

***Your implementation***
medI

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Medini-Rajendra, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Please also consider:
- [OPTIONAL] Please make sure you delete unnecessary files from your repository. It make your repository to look dirty. I can see you have two html files, two css files, one file with .orig(index.html.orig) extension, please check the ones that are not necessary and delete them. 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Medini-Rajendra) [reply index.html:43]: Thank you for your review. Since this is optional comment, I will implement in my future work.

File: index.html (hunk 2 above)
Comment (Medini-Rajendra) [reply index.html:138]: I have corrected this design issue in my code. Thank you.

Comment (Medini-Rajendra) [reply index.html:138]: I have also deleted unnecessary files from my repository as per the reviewer's suggestion. Thank you.

**Review by HillaryOkello — APPROVED** [review]
# Hi @Medini-Rajendra , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carlos-ssh** [comment]
Hi @Medini-Rajendra , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is missing a Pull Request Link open, this will make possible a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**